### PR TITLE
Add instructions for installing Atom using Linux package repos

### PR DESCRIPTION
**Pull request by daviwil**
I've updated the Linux installation instructions to add steps for configuring package repositories for distributions using the .deb and .rpm formats.  Would love for @ungb to double-check the steps before we merge this just to make sure everything works correctly :)

### CODE REVIEW

File: content/getting-started/sections/installing-atom.md [hunk 1]
```diff
@@ -86,25 +105,55 @@ $ sudo dpkg -i atom-amd64.deb
 $ sudo apt-get -f install
 ```
 
-##### RedHat and CentOS (yum)
+##### RedHat and CentOS (yum) or Fedora (dnf)
```
Comment (daviwil) [inline content/getting-started/sections/installing-atom.md]: Is it OK to merge these sections since the instructions are very similar?

File: content/getting-started/sections/installing-atom.md [hunk 2]
```diff
@@ -70,13 +70,32 @@ With Atom open, click on `File > Settings`, and then the `System` tab on the lef
...
+
+```
+# Install Atom
+sudo apt-get install atom
+
+# Install Atom Beta
+sudo apt-get install atom-beta
+```
+
+You can also download the Atom .deb package and install it directly:
```
Comment (daviwil) [inline content/getting-started/sections/installing-atom.md]: Should I say "Alternatively, you can..."?

File: content/getting-started/sections/installing-atom.md [hunk 3]
```diff
@@ -70,13 +70,32 @@ With Atom open, click on `File > Settings`, and then the `System` tab on the lef
...
 [releases]: https://github.com/atom/atom/releases/latest
 
 ##### Debian and Ubuntu (deb/apt)
 
-To install Atom on Debian, Ubuntu, or related systems:
+To install Atom on Debian, Ubuntu, or related distributions, add our official
+package repository to your system by running the following commands:
+
+```
+curl -L https://packagecloud.io/AtomEditor/atom/gpgkey | sudo apt-key add -
```
Comment (daviwil) [inline content/getting-started/sections/installing-atom.md]: There seems to be a convention of adding `$` in front of command lines in this document but I didn't use it in these blocks so it's easier to copy/paste into a terminal.  What do you think?

Comment (lee-dohm) [reply content/getting-started/sections/installing-atom.md]: Command line sections should be done with:

``````
``` command-line
$ command <em>replaceable parameter</em>
```
``````

The `$` gets converted to `C:\>` on Windows and the prompt isn't selectable. It's magic :sparkles:

File: content/getting-started/sections/installing-atom.md (hunk 2 above)
Comment (lee-dohm) [reply content/getting-started/sections/installing-atom.md]: Yes, that works :+1:

File: content/getting-started/sections/installing-atom.md (hunk 1 above)
Comment (lee-dohm) [reply content/getting-started/sections/installing-atom.md]: Yes, it appears to be clear enough.

File: content/getting-started/sections/installing-atom.md [hunk 4]
```diff
@@ -70,13 +70,32 @@ With Atom open, click on `File > Settings`, and then the `System` tab on the lef
...
+sudo sh -c 'echo "deb [arch=amd64] deb https://packagecloud.io/AtomEditor/atom/any/ any main" > /etc/apt/sources.list.d/atom.list'
+sudo apt-get update
```
Comment (50Wliu) [inline content/getting-started/sections/installing-atom.md]: I believe it's now recommended to use `apt` as a more beginner-friendly alternative, at least on Ubuntu.  Same for `apt install`.

Comment (daviwil) [reply content/getting-started/sections/installing-atom.md]: Thanks for pointing that out!  I think `apt` may only be available on Ubuntu variants and not Debian, so it may be better to use `apt-get` since it's the most universal command.  I'll mention that `apt` can also be used

**Comment by daviwil** [comment]
Updated the file to fix the command blocks and fixed an issue with the Debian repo configuration thanks to @ungb's help!

File: content/getting-started/sections/installing-atom.md [hunk 5]
```diff
@@ -70,13 +70,32 @@ With Atom open, click on `File > Settings`, and then the `System` tab on the lef
 
 #### Installing Atom on Linux
 
-To install Atom on Linux, you can download a [Debian package](https://atom.io/download/deb) or [RPM package](https://atom.io/download/rpm) either from the [main Atom website](https://atom.io) or from the [Atom project releases page][releases]. These packages do not currently have auto-update features, so when you would like to upgrade to a new release of Atom, you will have to repeat this installation process.
+To install Atom on Linux, you can download a [Debian package](https://atom.io/download/deb) or [RPM package](https://atom.io/download/rpm) either from the [main Atom website](https://atom.io) or from the [Atom project releases page][releases]. If you'd like Atom to be updated when new releases are published, you can configure an official package repository for your distribution.
```
Comment (ungb) [inline content/getting-started/sections/installing-atom.md]: Hmm I find it weird that we tell them they can download from the main site but now that the instructions are more towards installing using the package repository.  I wonder if we should remove the section about being able to download the builds or maybe have a separate section for manually installing.

Comment (daviwil) [reply content/getting-started/sections/installing-atom.md]: Yeah, I thought about that too.  Since I provide the alternative of installing the packages individually (with links to the download link in each distro section), I can remove these links from the top.  How about this wording instead:

> You can install Atom on Linux using your distribution's package manager by configuring it to use one of our official package repositories.  This will also allow you to update Atom when new releases are published.

Comment (ungb) [reply content/getting-started/sections/installing-atom.md]: Sounds good!  Should we include a section about how to manually install too?  I guess we shouldn't since this is way better.

Comment (daviwil) [reply content/getting-started/sections/installing-atom.md]: The manual install bits are still there, check out these sections, just search for the word "Alternatively" and you'll see the manual steps for the 3 package managers

**Comment by daviwil** [comment]
Thanks for all the feedback!  Merging this now.  If you think of any other changes I should make I'll be happy to open another PR.
